bundles/org.openhab.binding.miio/README.base.md

```diff
@@ -240,6 +240,8 @@ e.g. `openhab:send actionCommand 'upd_timer["1498595904821", "on"]'` would enabl
 | String  | info#fw_features                  | Firmware Features          |
 | String  | info#room_mapping                 | Room Mapping details       |
 | String  | info#multi_maps_list              | Maps Listing details       |
+| Number  | status#mop_drying_time            | Mop Drying Time            |
+| Switch  | status#is_mop_drying              | Mop cleaning active        |
```

Comment:
suggest to move these also to the section about the additional depending on the capabilities, as these are defiantly not available on all robots

Reply:
I misses that. 

I moved them underneath the additional capabilities